docker-compose.yml

```diff
@@ -11,7 +11,9 @@ services:
       - ../.cache/huggingface/:/root/.cache/huggingface/
       - ./static/:/app/static/
     ports:
-      - "8000:8000"
+      - "7500:7500"
+      - "7501:7501"
```

Comment:
what are these for

Reply:
port in docker for serving frontend, backend app

Reply:
I need to give an open port to EVAL, but I'm wondering how to give it. 1. order to find the open port directly, 2. tell the open port at the system prompt, 3. tell the user prompt.